app.js

```diff
@@ -8,13 +8,23 @@ const app = express();
 app.use(express.json());
 
 app.get('/api/topics', getTopics);
+app.get('/api/articles/:article_id', getArticleById);
 
 app.use((err, req, res, next) => {
-  // ready for err handling.
-})
-
+  const badRequestCodes = [];
+  if (badRequestCodes.includes(err.code)) {
+    res.status(400).send({ message: 'bad request.'});
+  } else {
+    next(err);
+  };
+});
 
 
+// handles unexpected errors - check test log for error codes
+app.use((err, req, res, next) => {
+  console.log(err, '<-- error');
```

Comment:
Think about whether you still need this console.log - if not, remove it to keep your code clean

Reply:
removing!